src/Stratis.Bitcoin/Configuration/TextFileConfiguration.cs

```diff
@@ -76,7 +80,11 @@ public TextFileConfiguration(string data)
                     throw new FormatException("Line " + lineNumber + $": \"{l}\" : No value is set");
 
                 // Add to dictionary. Trim spaces around keys and values.
-                this.Add(split[0].Trim(), string.Join("=", split.Skip(1)).Trim());
+                string key = split[0].Trim();
+                if (!key.StartsWith("-"))
```

Comment:
Why add a dash? To make all the keys more homogeneous?

Reply:
Yes. The keys don't have a dash in the configuration file while they do include a dash when passed via the command-line. Adding the dash here makes the keys homogeneous.